pwnlib/util/packing.py

```diff
@@ -80,23 +80,23 @@ def pack(number, word_size = None, endianness = None, sign = None, **kwargs):
         >>> pack(0x414243, 24, 'little', True)
         b'CBA'
         >>> pack(0x814243, 24, 'big', False)
-        b'\\x81BC'
+        b'\x81BC'
```

Comment:
It seems like these tests should have been failing -- are we sure they're being run?

Reply:
See the `r"""` change above